paper.md

```diff
@@ -56,7 +56,7 @@ We believe that the large renewed interest in LISA and the stellar-origin source
 
 \lw{} is a Python package designed to streamline the process of making predictions of LISA detection rates for stellar-origin binaries such that it is as fast, reliable and simple as possible. This goal makes \lw{} unique among other implementations of gravitational-wave tools in the literature, which focus on a more broad coverage of the gravitational-wave spectrum and source classes, rather than an optimised approach for certain sources (e.g. @Moore+2015; @Yi2021). With \lw{} one can evolve the orbits of a binary or a collection of binaries and calculate their strain amplitudes for any range of frequency harmonics. One can compute the sensitivity curve for LISA or other future gravitational wave detectors (e.g. TianQin [@Luo+2016], or a user-defined custom instrument) and use it to compute the signal-to-noise ratio of a collection of sources. Furthermore, \lw{} provides tools to visualise all of the results with easy-to-use plotting functions. Finally, \lw{} is fully tested to check for consistency in the derivations described in the ApJ paper [@Wagg+2021].
 
-Specifically, we implement the expressions by @Peters+1963 and @Peters+1964 for the evolution of binary orbits due to the emission of gravitational waves, equations for the strain amplitudes and signal-to-noise ratios of binaries from various papers (e.g. @Flanagan+1998; @Finn+2000; @Cornish2003; e.g. @Barack+2004; @Moore+2015) and approximations for the LISA and TianQin sensitivity curves given in @Robson+2019 and @Huang+2020 respectively.
+Specifically, we implement the expressions by @Peters+1963 and @Peters+1964 for the evolution of binary orbits due to the emission of gravitational waves, equations for the strain amplitudes and signal-to-noise ratios of binaries from various papers (e.g. @Flanagan+1998; @Finn+2000; @Cornish2003; e.g. @Barack+2004; @Moore+2015) and approximations for the LISA and TianQin sensitivity curves given in @Robson+2019 and @Huang+2020 respectively. This lowest-order post-Newtonian implementation of gravitational radiation does not include the effects of higher-order terms in the post-Newtonian expansion, including spin-orbit coupling or radiation reaction. The strain calculations in \lw{} also incorporate averages over the detector motion as well as sky position and inclination in order to remain completely analytically solvable. Work which requires the incorporation of a full LISA detector simulation should consider tools like lisacattools or ldasoft (@Littenberg+2020). For non-stellar-origin sources, tools like the Gravitational Wave Universe Toolbox are more appropriate (@Yi+2021). 
```

Comment:
Noticed an extra e.g., changed the syntax for bracketed single citations and switched to the original Yi bibtex entry

```suggestion
Specifically, we implement the expressions by @Peters+1963 and @Peters+1964 for the evolution of binary orbits due to the emission of gravitational waves, equations for the strain amplitudes and signal-to-noise ratios of binaries from various papers (e.g. @Flanagan+1998; @Finn+2000; @Cornish2003; @Barack+2004; @Moore+2015) and approximations for the LISA and TianQin sensitivity curves given in @Robson+2019 and @Huang+2020 respectively. This lowest-order post-Newtonian implementation of gravitational radiation does not include the effects of higher-order terms in the post-Newtonian expansion, including spin-orbit coupling or radiation reaction. The strain calculations in \lw{} also incorporate averages over the detector motion as well as sky position and inclination in order to remain completely analytically solvable. Work which requires the incorporation of a full LISA detector simulation should consider tools like lisacattools or ldasoft [@Littenberg+2020]. For non-stellar-origin sources, tools like the Gravitational Wave Universe Toolbox are more appropriate [@Yi2021]. 
```